vipps-ecom-api-faq.md

```diff
@@ -853,9 +854,44 @@ to the requirements.
 
 There is no need for any changes to your Vipps implementation for SCA.
 
+### How can I use Vipps for different types of payments?
+
+It's possible to use the Vipps eCom API for several different types of payments.
+
+Let's say you run a bookstore. You can then Vipps eCom API in different ways:
+
+1. A webshop that sells physical books:
+   Vipps eCom API with "reserve capture", since you can not capture the payment before the book is shipped.
+2. A webshop that sells digital, downloadable books that are immediately available:
+   Vipps eCom API with "direct capture".
+3. A physical store where customers buy physical books in person:
+   Vipps eCom API with "reserve capture", possibly integrated with the POS.
```

Comment:
should this be direct?

Reply:
Do you mean item 3? Yes, it shoyld probably be. Changing.